http3/roundtrip.go

```diff
@@ -245,3 +260,13 @@ func validMethod(method string) bool {
 func isNotToken(r rune) bool {
 	return !httpguts.IsTokenRune(r)
 }
+
+func makeDialer(udpConn *net.UDPConn) func(ctx context.Context, addr string, tlsCfg *tls.Config, cfg *quic.Config) (quic.EarlyConnection, error) {
```

Comment:
Would it make sense to have this as a method on the roundtripper?

Reply:
It does make sense. In this way we don't need to pass a `*net.UDPConn` to it, it can use the `udpConn` inside the `RoundTripper` directly.